src/main/java/com/yetanalytics/xapi/model/Attachment.java

```diff
@@ -14,6 +18,7 @@ public class Attachment {
     private String sha2;
     private String fileUrl;
 
+    
```

Comment:
Extra space?

Reply:
fixed (59646edfc8cb2d3de629b961452f5aa15c75c4ac) but also nobody cares in java.